spyder/plugins/editor/widgets/editor.py

```diff
@@ -434,6 +443,9 @@ class EditorStack(QWidget):
     sig_go_to_definition = Signal(str, int, int)
     perform_lsp_request = Signal(str, str, dict)
     sig_option_changed = Signal(str, object)  # config option needs changing
+    save_bookmark = Signal(int)
+    load_bookmark = Signal(int)
+    save_bookmarks = Signal(str, str)

```

Comment:
@dalthviz, please change these signals to `sig_save_bookmark`, `sig_load_bookmark` and `sig_save_bookmarks`.